server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -473,6 +473,23 @@ Response codes:
 
 This also can be used to create a new user.
 
+
+=== Verifying a user password
+
+....
+curl -XPOST http://ip:port/users/usernameToBeUsed \
```

Comment:
Should we have a new path for this? 
IMO, `users/usernameToBeUsed` looks like a common path
Ex: 
- `/users/usernameToBeUsed/authen-check`
- `/users/authen-check`
- ...

Reply:
Good idea, it makes the intention more clear, and avoids unfortunate accidents if someone confuses POST and PUT. I will stick with "verify" as action verb though, as authentication is more associated with JWT generation.